File: ce/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples1.vb
' <snippetlinqexamples1>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_where1 = From c In svcContext.AccountSet _
                     Where c.Name.Contains("Contoso") _
                     Select c

  For Each c In query_where1
   Console.WriteLine(c.Name &amp; " " &amp; c.Address1_City)
  Next c
 End Using

' </snippetlinqexamples1>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples1.cs
// <snippetlinqexamples1>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_where1 = from a in svcContext.AccountSet
                     where a.Name.Contains("Contoso")
                     select a;
  foreach (var a in query_where1)
  {
   System.Console.WriteLine(a.Name + " " + a.Address1_City);
  }
 }

// </snippetlinqexamples1>